src/backend/src/services/work-packages.services.ts

```diff
@@ -464,33 +453,35 @@ export default class WorkPackagesService {
         stage,
         blockedBy: {
           set: [], // remove all the connections then add all the given ones
-          connect: depsIds.map((ele) => ({ wbsElementId: ele }))
+          connect: updatedBlockedBys.map((ele) => ({ wbsElementId: ele.wbsElementId }))
         }
       }
     });
 
     // Update any deleted description bullets to have their date deleted as right now
-    const deletedIds = expectedActivitiesChangeJson.deletedIds.concat(deliverablesChangeJson.deletedIds);
-    if (deletedIds.length > 0) {
+    const deletedElements: DescriptionBulletPreview[] = expectedActivitiesChangeJson.deletedElements.concat(
+      deliverablesChangeJson.deletedElements
+    );
+    if (deletedElements.length > 0) {
       await prisma.description_Bullet.updateMany({
-        where: { descriptionId: { in: deletedIds } },
+        where: { descriptionId: { in: deletedElements.map((descriptionBullet) => descriptionBullet.id) } },
         data: { dateDeleted: new Date() }
       });
     }
 
-    addDescriptionBullets(
-      expectedActivitiesChangeJson.addedDetails,
+    await addDescriptionBullets(
+      expectedActivitiesChangeJson.addedElements.map((descriptionBullet) => descriptionBullet.detail),
       updatedWorkPackage.workPackageId,
```

Comment:
how can we pass in a workPackageId here but a projectId in the project services file?